### ISSUE TITLE
Create Threshold from Template --> uncaught PHP error

### ISSUE THREAD

**Issue by dsclassen** [Cacti/plugin_thold#233]
When I attempt to create a threshold from an existing graph I receive the following error:

This also disables the entire hold plugin.

FYI
I am running cacti 1.2.0
hold from GitHub 7914628
php 7.1.25
on CentOS 7

```
2019-01-07 09:19:28 - MAILER ERROR: Invalid address: (to): Administrator
2019-01-07 09:19:28 - CMDPHP ERRORS DETECTED - DISABLING PLUGIN 'thold'
2019-01-07 09:19:28 - CMDPHP PHP ERROR Backtrace: (CactiShutdownHandler())
2019-01-07 09:19:28 - ERROR PHP ERROR in Plugin 'thold': Uncaught Error: [] operator not supported for strings in /var/www/cacti/plugins/thold/setup.php:1007 Stack trace: #0 /var/www/cacti/lib/plugins.php(221): thold_graphs_action_prepare(Array) #1 /var/www/cacti/lib/plugins.php(114): api_plugin_run_plugin_hook_function('graphs_action_p...', 'thold', 'thold_graphs_ac...', Array) #2 /var/www/cacti/graphs.php(1216): api_plugin_hook_function('graphs_action_p...', Array) #3 /var/www/cacti/graphs.php(80): form_actions() #4 {main} thrown in file: /var/www/cacti/plugins/thold/setup.php on line: 1007
```

**Comment by three6five-kzn** [comment]
I actually just created a pull request to fix this a few hours before you posted this.

https://github.com/Cacti/plugin_thold/pull/231

tldr; You need to modify line line 986 on setup.php to read:

>$template_ids = [];

instead of:

>$template_ids = '';

Hope that helps!

**Comment by dsclassen** [comment]
it helped indeed. Thanks.

**Comment by TheWitness** [comment]
@dsclassen and @three6five-kzn.  Fixed just a little bit differently.  Thanks for testing.
